go/oasis-node/cmd/debug/txsource/workload/transfer.go

```diff
@@ -21,10 +21,9 @@ const (
 	// NameTransfer is the name of the transfer workload.
 	NameTransfer = "transfer"
 
-	transferNumAccounts  = 10
-	transferAmount       = 1
-	transferFundInterval = 10
-	transferGasCost      = 10
```

Comment:
:information_source: the fate of this is that the new helper does a proper gas estimation for each transaction